### ISSUE TITLE
Bro you had src there wasnt anything to crack just remove a few lines

### ISSUE THREAD

**Issue by vasler4** [CatGirlLana/OnyxRecodeCracked#1]

**Comment by Ceptea** [comment]
real

**Comment by PlayingCow61105** [comment]
Yeah but Lana is a dumbass so all she does is remove a few lines, make it look like she put in work, then start being an absolute fucking cock to everyone. This truly shows why abortion should be mandatory for some people.

**Comment by CatGirlLana** [comment]
Had used recaf to remove the auth in some jars like updated beta jars etc not just src. That's why src isn't fully updated

**Comment by integr-dev** [comment]
bro did not do anything lmao

**Comment by integr-dev** [comment]
bro got given all the source

**Comment by PlayingCow61105** [comment]
I'm literally cumming in laughter seeing this so far into the future. This is fucking sad. Also, saying I leaked the jar, then editing a photo of someone leaking the jar into me leaking it, is fucking laughable.
